### PR TITLE
Sequelize Migrations Working

### PR DESCRIPTION
**Pull request by hernan-dp**

### CODE REVIEW

File: src/migrations/20200121111445-create-user.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    return queryInterface.createTable('user', {
+      id: {
+        type: Sequelize.UUID,
+        defaultValue: Sequelize.UUIDV4,
+        allowNull: false,
+        primaryKey: true
+      },
+      firstName: {
+        type: Sequelize.STRING,
+        allowNull: true
```
Comment (rdemasi) [inline src/migrations/20200121111445-create-user.js]: firstName and lastName should be allowNull false here and also on the model

File: src/models/user.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  }
+
+  User.encryptPassword = function (plainPassword, salt) {
+    return crypto.scryptSync(plainPassword, salt, 64).toString('hex')
+  }
+   // hooks
+   User.beforeCreate(User.hashPasswordHook.bind(User))
+   User.beforeUpdate(User.hashPasswordHook.bind(User))
+  return User;
+};
```
Comment (rdemasi) [inline src/models/user.js]: Add new line here

File: src/models/user.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  const User = sequelize.define('User', {
+    firstName: DataTypes.STRING,
+    lastName: DataTypes.STRING,
+    username: DataTypes.STRING,
+    salt: DataTypes.STRING,
+    password: DataTypes.STRING
+  }, {});
+  User.associate = function(models) {
+    // associations can be defined here
+  };
```
Comment (rdemasi) [inline src/models/user.js]: remove semicolon here. Please, run standarjs

**Review by rdemasi — CHANGES_REQUESTED** [review]

**Review by nahudalla — APPROVED** [review]

**Review by rdemasi — APPROVED** [review]
LGTM

**Pull request merged**